### PR TITLE
Cleanup dcp1 terra script

### PR DESCRIPTION
**Pull request by jacobwindsor**
Adds a script to cleanup staging areas of dcp1 projects

### CODE REVIEW

File: scripts/cleanup-dcp1/readme.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Cleanup DCP1 projects
+This script cleans any the terra staging area of DCP1 projects so that they are ready to export again if changes have been made
+
+**You need to have gsutil setup with the correct credentials prior to running this**
```
Comment (aaclan-ebi) [inline scripts/cleanup-dcp1/readme.md:4]: could link to the SOP in setting up terra access

**Review by aaclan-ebi — APPROVED** [review]

File: scripts/cleanup-dcp1/readme.md (hunk 1 above)
Comment (MightyAx) [reply scripts/cleanup-dcp1/readme.md:4]: please put the link here as I have to do this tomorrow.

**Review by amnonkhen — APPROVED** [review]

**Review by ke4 — APPROVED** [review]
Just add a tiny comment, but looks fine by me 💯

File: scripts/cleanup-dcp1/cleanup.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+                skipped.append((uuid, f'Not DCP1. Project ID {project_uuid}'))
+                continue
+            
+            logger.info(f'project {project_uuid} is a DCP1 project')
+
+            if not input('Would you like to continue with fixing the terra area for this submission? (Y/n)').lower() == 'y':
+                skipped.append((uuid, 'Manually skipped'))
+                continue
+
+            utils.run_command(f'gsutil -m rm -r {GCP_BUCKET}/{project_uuid}/metadata/sequence_file')
```
[the remark this answers is not in the document]
Comment (jacobwindsor) [reply scripts/cleanup-dcp1/cleanup.py:40]: It gets logged to the csv

File: scripts/cleanup-dcp1/utils.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+    return Submission(sub['_links']['self']['href'], INGEST_API_TOKEN)
+
+def run_command(command: str):
+    process = subprocess.Popen(command.split(), stdout=subprocess.PIPE)
+    output, error = process.communicate()
+    
+    if error:
+        raise Exception(f'Command failed: {error}')
+    return output
```
[the remark this answers is not in the document]
Comment (jacobwindsor) [reply scripts/cleanup-dcp1/utils.py:34]: You can see the output in stdout. I don't think any extra logging is necessary. If you want you can just pipe it somewhere

File: scripts/cleanup-dcp1/readme.md (hunk 1 above)
Comment (jacobwindsor) [reply scripts/cleanup-dcp1/readme.md:4]: https://ebi-ait.github.io/hca-ebi-dev-team/admin_setup/Setting-up-access-to-Terra-staging-area.html

**Pull request merged**